Tests/TestCreateFFmpegInteropMSSFromStream.cs

```diff
@@ -30,216 +31,170 @@ namespace UnitTest.Windows
-            Assert.AreEqual(FFmpegMSS.AudioCodecName.ToLowerInvariant(), "aac");
-            Assert.AreEqual(FFmpegMSS.VideoCodecName.ToLowerInvariant(), "h264");
-
-            MediaStreamSource mss = FFmpegMSS.GetMediaStreamSource();
-            Assert.IsNotNull(mss);
-
-            // Based on the provided media, check if the following properties are set correctly
-            Assert.AreEqual(true, mss.CanSeek);
-            Assert.AreNotEqual(0, mss.BufferTime.TotalMilliseconds);
-            Assert.AreEqual(Constants.DownloadUriLength, mss.Duration.TotalMilliseconds);
-        }
-
-        [TestMethod]
-        public async Task CreateFromStream_Destructor()
-        {
-            Uri uri = new Uri(Constants.DownloadUriSource);
-            Assert.IsNotNull(uri);
-
-            StorageFile file = await StorageFile.CreateStreamedFileFromUriAsync(Constants.DownloadStreamedFileName, uri, null);
-            Assert.IsNotNull(file);
-
-            IRandomAccessStream readStream = await file.OpenAsync(FileAccessMode.Read);
-            Assert.IsNotNull(readStream);
-
-            // CreateFFmpegInteropMSSFromStream should return valid FFmpegInteropMSS object which generates valid MediaStreamSource object
-            FFmpegInteropMSS FFmpegMSS = FFmpegInteropMSS.CreateFFmpegInteropMSSFromStream(readStream, false, false);
-            Assert.IsNotNull(FFmpegMSS);
-
-            // Validate the metadata
-            Assert.AreEqual(FFmpegMSS.AudioCodecName.ToLowerInvariant(), "aac");
-            Assert.AreEqual(FFmpegMSS.VideoCodecName.ToLowerInvariant(), "h264");
-
-            MediaStreamSource mss = FFmpegMSS.GetMediaStreamSource();
-            Assert.IsNotNull(mss);
+            MediaStreamSource mss = CreateMSSFromStream(stream, config);
 
             // Based on the provided media, check if the following properties are set correctly
-            Assert.AreEqual(true, mss.CanSeek);
-            Assert.AreNotEqual(0, mss.BufferTime.TotalMilliseconds);
+            Assert.IsTrue(mss.CanSeek);
+            Assert.IsTrue(mss.BufferTime.TotalMilliseconds > 0);
             Assert.AreEqual(Constants.DownloadUriLength, mss.Duration.TotalMilliseconds);
 
-            // Keep original reference and ensure object are not destroyed until each reference is released by setting it to nullptr
-            FFmpegInteropMSS OriginalFFmpegMSS = FFmpegMSS;
-            MediaStreamSource Originalmss = mss;
-
-            FFmpegMSS = FFmpegInteropMSS.CreateFFmpegInteropMSSFromStream(null, false, false);
-            Assert.IsNull(FFmpegMSS);
-            Assert.IsNotNull(OriginalFFmpegMSS);
-            Assert.IsNotNull(Originalmss);
-
-            mss = null;
-            Assert.IsNull(mss);
-            Assert.IsNotNull(Originalmss);
-
-            OriginalFFmpegMSS = null;
-            Originalmss = null;
-            Assert.IsNull(OriginalFFmpegMSS);
-            Assert.IsNull(Originalmss);
+            // TODO: Verify expected stream types (aac, h264)
```

Comment:
Add explanation about future change or remove

Reply:
I'll handle test updates in a future change.